src/components/FieldPreview/FieldPreview.stories.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import {
+  withKnobs,
+  text,
+  boolean,
+} from '@storybook/addon-knobs'
+
+import { FieldPreview } from 'components/FieldPreview/FieldPreview'
+
+const PRIVATE_KEY = '0xfb27c2394586feb01403ba3643b519c8f209e0427b31f510a6877b494c020f59'
```

Comment:
It's a public key :)

Reply:
Yeah, I know, I just copy this line from other story